### DISCUSSION TITLE
Confluence V2 REST API Compatibility

### DISCUSSION THREAD

**Discussion by havalotatime** [docToolchain/docToolchain#1191] [Q&A]
Greetings! We have been looking at adopting docToolChain and integrating the docs as code way of life into our development philosophy. One big plus was the ability to publish our technical documents to confluence as part of our build pipeline(s) alongside our code. The benefits are plentiful and this way of working makes perfect sense. 
Confluence have deprecated quite a large amount of the V1 REST API (used by the docToolChain framework), which is going to break anything that currently makes use of these processes when they stop the V1 API at the end of year (if memory serves)
I didn't find anything in your space referencing an upgrade to the V2 Confluence REST API - is this something that will get any attention any time soon?

**Comment by PacoVK** [comment]
Hey @havalotatime we already had this on our desk, but there wasn't (yet) an issue to track. I just created #1205 for further discussion on that topic and for reference.
